receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper.go

```diff
@@ -233,6 +242,10 @@ func (s *scraper) getProcessMetadata() ([]*processMetadata, error) {
 		data = append(data, md)
 	}
 
+  if err := s.refreshHandleCounts(); err != nil {
```

Comment:
```suggestion
	if err := s.refreshHandleCounts(); err != nil {
```

Reply:
Also, wouldn't the first `scrape` always be empty since `refreshHandleCount` is called after `scrapeAndAppendHandlesMetric`?